apps/pkcs12.c

```diff
@@ -84,7 +84,8 @@ const OPTIONS pkcs12_options[] = {
     {"certpbe", OPT_CERTPBE, 's', "Certificate PBE algorithm (default 3DES)"},
 # endif
     {"export", OPT_EXPORT, '-', "Output PKCS12 file"},
-    {"noiter", OPT_NOITER, '-', "Don't use encryption iteration"},
+    {"iter", OPT_ITER, 'p', "Specify the encryption key iteration count"},
+    {"noiter", OPT_NOITER, '-', "Don't use encryption key iteration"},
```

Comment:
Should we disallow specifying both "-iter" and "-noiter" in combination? It makes no sense to allow it.

Reply:
It's already possible to specify multiple `-iter` options (in `pkcs8` command for example), and the last one takes efffect. `-noiter` is simply an alias for `-iter 1` so I think it's fine to allow both.